packages/engine/lib/simulation-control/src/tests/arrow.rs

```diff
@@ -55,30 +55,24 @@ fn get_schema() -> Result<()> {
 
     field_spec_map.try_extend([root_field_spec_from_agent_field(AgentStateField::AgentId)?])?;
 
-    let mut meta = HashMap::new();
+    let mut meta = BTreeMap::new();
```

Comment:
Any specific reason why choosing `BTreeMap` over `HashMap` here?

Reply:
That's good enough I guess. Unless we do micro-optimizations it shouldn't matter at all